app/assets/stylesheets/common/components/buttons.scss

```diff
@@ -27,6 +27,10 @@
     color: $icon-color;
     margin-right: 0.45em;
     transition: color 0.25s;
+
+    @media (forced-colors: active) {
```

Comment:
Does this media query apply to WHCM only? If so, it might be cleaner fo have a separate file with just the WHCM changes.

Reply:
Good point, I'll push out a change now.

Reply:
I think the suggestion was to move all those `@media` queries to that file 😃

Reply:
This one needs to be declared inside the mixin, but I've added comments for more clarity.